src/enforcer/HolographRoyalties.sol

```diff
@@ -297,18 +297,13 @@ contract HolographRoyalties is Admin, Owner, Initializable {
     address payable[] memory addresses = _getPayoutAddresses();
     uint256[] memory bps = _getPayoutBps();
     uint256 length = addresses.length;
-    // accommodating the 2300 gas stipend
-    // adding 1x for each item in array to accomodate rounding errors
-    uint256 gasCost = (23300 * length) + length;
     uint256 balance = address(this).balance;
-    require(balance - gasCost > 10000, "ROYALTIES: Not enough ETH");
-    balance = balance - gasCost;
+    require(balance > 10000, "ROYALTIES: Not enough ETH");
     uint256 sending;
-    // uint256 sent;
     for (uint256 i = 0; i < length; i++) {
```

Comment:
Multiple wardens suggested we move payments to a pull instead of push approach. That would require more changes and likely keeping a mapping of payees. Before moving forward with that I'd like to get your input Vitto https://github.com/code-423n4/2022-10-holograph-findings/issues/104

Reply:
Our goal with the HolographRoyalties contract is to only collect the royalties and provide a way to access the amounts after. If only one payout recipient is configured, this could be considered as a pull approach. The push approach is set as the default/only option simply due to the fact that we do not plan to add any features/functionality outside of the scope of collecting the royalties. Any additional distribution logic should be handled by an external contract code.